src/requirements/data/index.ts

```diff
@@ -77,41 +77,50 @@ const json: RequirementsJson = {
     UNI: {
       name: 'University',
       requirements: universityRequirements,
+      abbr: 'other',
     },
   },
   college: {
     AG: {
       name: 'Agriculture and Life Sciences',
       requirements: calsRequirements,
+      abbr: 'cals',
     },
     AR: {
       name: 'Architecture, Art and Planning',
       requirements: aapRequirements,
+      abbr: 'aap',
     },
     AS1: {
       name: 'Arts and Sciences [before Fall 2020]',
       requirements: casPreFA2020Requirements,
+      abbr: 'a&s',
```

Comment:
Can you change this to `cas`?

Reply:
:( i do like a&s but since that's the general consensus I have changed it to cas